packages/react-composites/src/composites/common/Drawer/MoreDrawer.tsx

```diff
@@ -349,6 +374,7 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
     itemKey: 'defaultSelectionKey',
     text: localeStrings.strings.call.moreButtonGalleryDefaultLayoutLabel,
     onItemClick: () => {
+      /* @conditional-compile-remove(gallery-layout-composite) */
```

Comment:
this needs to stay for the composite

Reply:
But there are other parts where we have cc ... those were not removed. so it ends up bubbling up errors. For the sake of successful I am adding this. Removing them should be a different KTLO item.

Reply:
In the MoreDrawer File there are 10 other occurences which was there and I did not add or remove them. Without adding these 4, I get build error for stable. Lets make a separate task to remove those CC.